<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>player</class>
 <widget class="QDialog" name="player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="n_player_1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="n_player_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="n_player_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="n_player_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Annulla</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>551</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Giocatori</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>72</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="player_1">
      <property name="text">
       <string>Player 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="player_2">
      <property name="text">
       <string>Player 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="player_3">
      <property name="text">
       <string>Player 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="player_4">
      <property name="text">
       <string>Player 4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>player</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>player_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>n_player_1</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>player_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>n_player_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>player_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>n_player_3</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>player_4</sender>
   <signal>toggled(bool)</signal>
   <receiver>n_player_4</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
